src/parse-changelog.ts

```diff
@@ -30,23 +30,37 @@ function isValidChangeCategory(category: string): category is ChangeCategory {
  * @param options - Options.
  * @param options.changelogContent - The changelog to parse.
  * @param options.repoUrl - The GitHub repository URL for the current project.
- * @param options.tagPrefix - The prefix used in tags before the version number.
- * @param options.formatter - A custom Markdown formatter to use.
+ * @param [options.tagPrefix] - The prefix used in tags before the version number.
+ * @param [options.formatter] - A custom Markdown formatter to use.
+ * @param [options.versionBeforePackageRename] - A version string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
+ * @param [options.tagPrefixBeforePackageRename] - A tag prefix string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
  * @returns A changelog instance that reflects the changelog text provided.
  */
 export function parseChangelog({
   changelogContent,
   repoUrl,
   tagPrefix = 'v',
   formatter = undefined,
+  versionBeforePackageRename = undefined,
+  tagPrefixBeforePackageRename = undefined,
 }: {
   changelogContent: string;
   repoUrl: string;
   tagPrefix?: string;
   formatter?: Formatter;
+  versionBeforePackageRename?: string;
+  tagPrefixBeforePackageRename?: string;
 }) {
   const changelogLines = changelogContent.split('\n');
-  const changelog = new Changelog({ repoUrl, tagPrefix, formatter });
+  const changelog = new Changelog({
```

Comment:
Since it would now be possible to change the behavior by passing these options, what do you think about adding a new test which verifies that a changelog that features renamed tags can be successfully parsed?

Reply:
It looks like the test you pointed to covers the changes you've made to `validateChangelog`. However, `parseChangelog` is used in both `validateChangelog` and `updateChangelog`, and so I think it'd be valuable for us to confirm that it works well when called on its own. Does that make sense?